data/constants.js

```diff
@@ -1,2 +1,2 @@
+module.exports = { colorCode };
```

Comment:
its probably better to use esm modules here.  Just use commonjs for the knex stuff / other initialization files.